mappings/net/minecraft/block/PointedDripstoneBlock.mapping

```diff
@@ -8,6 +8,8 @@ CLASS net/minecraft/class_5689 net/minecraft/block/PointedDripstoneBlock
 	FIELD field_28056 FRUSTUM_SHAPE Lnet/minecraft/class_265;
 	FIELD field_28057 MIDDLE_SHAPE Lnet/minecraft/class_265;
 	FIELD field_28058 BASE_SHAPE Lnet/minecraft/class_265;
+	FIELD field_31211 MAX_WATER_DRIP_CHANCE F
```

Comment:
```suggestion
	FIELD field_31211 WATER_DRIP_CHANCE F
```
This is a chance itself. The `dripTick` method checks if a random number is less than `0.17578125`. The `dripChance` parameter is actually a sampled value.